src/mscorlib/src/System/Text/Normalization.Windows.cs

```diff
@@ -49,6 +56,13 @@ internal static bool IsNormalized(String strInput, NormalizationForm normForm)
 
         internal static String Normalize(String strInput, NormalizationForm normForm)
         {
+            if (GlobalizationMode.Invariant)
+            {
+                // In Invariant mode we assume all characters are normalized. 
```

Comment:
if there is any library using System.Globalization.Native.so will need to initialize the ICU manually as we stopped initializing it automatically